applications/yc_w16.md

```diff
@@ -0,0 +1,430 @@
+national average) and 37% are minorities in tech (7x industry average).
+
+**If you've already started working on it, how long have you been working and
+how many lines of code (if applicable) have you written?**
+
+We've been working on hackEDU full-time for 11 months.
+
+- Lines of code: 12,333 across 349 files (excludes 2113 blank lines)
+- Lines of Markdown: 31,199 across 348 files (148,653 words)
+
+Note: given the nature of our work, most of our development happens in Markdown
+files.
+
+**Which of the following best describes your progress?**
+
+- [ ] Nothing Built
+- [ ] Sketches
+- [ ] Mocks / Renderings
+- [ ] Prototype
+- [x] Private Beta
+- [ ] Public Beta
+- [ ] Taking Preorders
+- [ ] Launched
+
+**Do you have revenue?**
+
+- [ ] No
+- [x] Yes
+
+**How much revenue?**
+
+$91,715.00
+
+**If you've applied previously with the same idea, how much progress have you
+made since the last time you applied? Anything change?**
+
+We've expanded our reach by 11 schools and two countries, bringing our total
+reach to an estimated 1,500 students in 11 states and 4 countries. Our inbound
+funnel has also grown immensely with 36 more vetted school applications from 10
+states and 2 countries that we'll be onboarding starting January 2016.
+
+We've significantly increased our inbound funnel and have 36 vetted schools from
+10 states and 4 countries on our waitlist. Additionally we've expanded our reach
+by 11 schools and 2 countries, bringing our total reach to 47 schools across 11
+states and 4 countries, reaching an estimated 1,500 students.
+
+The biggest change since last applying is that we've clarified the
+implementation of our product---the club playbook---and have made significant
+strides towards creating it. We spent the summer testing different curriculum
+layouts until we found one that we were confident in and we've built that into
+our playbook. We're now piloting it at 12 schools this semester. _(Perhaps boil
+this down to "We're piloting the software and playbook we are developing with 12
+schools this semester.")_
+
+Before we focus on expansion we want to make sure we're creating clubs that
+people want to go to. We're planning on gradually starting clubs with the
+playbook from our waitlist starting January 2016.
+
+**If you have already participated or committed to participate in an incubator,
+"accelerator" or "pre-accelerator" program, please tell us about it.**
+
+We have not.
+
+**Why did you pick this idea to work on? Do you have domain expertise in this
+area? How do you know this is going to work?**
+
+In high school Zach started a coding club. By the end of its 2nd year, the
+members of his club launched games on the Android Play Store, won high-profile
+hackathons, founded a web design firm, and secured over $30,000 in investment.
+After meeting other students from around the country who also had started clubs,
+he decided to start hackEDU to scale the model.
```

Comment:
This does not match the level of success I saw with the club. Many of these achievements are personal rather than club oriented.

Also, presenting success as an individual running a coding club does not show the need for hackedu; I was under the impression hackedu was created because it is hard for an individual to create a club without support.